src/Native/Unix/System.Security.Cryptography.Native.Apple/pal_x509chain.c

```diff
@@ -264,3 +264,17 @@ int32_t AppleCryptoNative_GetOSStatusForChainStatus(PAL_X509ChainStatusFlags cha
             return errSecCoreFoundationUnknown;
     }
 }
+
+void AppleCryptoNative_X509ChainSetTrustAnchorCertificates(SecTrustRef chain, CFArrayRef anchorCertificates)
+{
+    if (chain == NULL)
+    {
+        return;
+    }
+    if (anchorCertificates == NULL)
+    {
+        return;
+    }
```

Comment:
Are null chains and anchorCertificates considered valid?  If not, should we be returning an error of some kind?